frontend/src/scenes/settings/environment/BounceRateDuration.tsx

```diff
@@ -0,0 +1,34 @@
+import { useActions, useValues } from 'kea'
+import { LemonButton } from 'lib/lemon-ui/LemonButton'
+import { LemonInput } from 'lib/lemon-ui/LemonInput'
+import { useState } from 'react'
+import { teamLogic } from 'scenes/teamLogic'
+
+export function BounceRateDurationSetting(): JSX.Element {
+    const { updateCurrentTeam } = useActions(teamLogic)
+    const { currentTeam } = useValues(teamLogic)
+
+    const savedDuration =
+        currentTeam?.modifiers?.bounceRateDurationSeconds ?? currentTeam?.default_modifiers?.bounceRateDurationSeconds
+    const [bounceRateDuration, setBounceRateDuration] = useState<number | undefined>(savedDuration)
+
+    const handleChange = (duration: number | undefined): void => {
+        updateCurrentTeam({ modifiers: { ...currentTeam?.modifiers, bounceRateDurationSeconds: duration } })
+    }
+
+    return (
+        <>
+            <p>Choose how long a user can stay on a page, before the session is not a bounce.</p>
```

Comment:
Let's make it clear we're talking seconds here

Reply:
Done